data-mermaid/mermaid-webapp
# Add fix to only show error specific to project

[trello card](https://trello.com/c/zFaued8v/990-i-like-the-warning-message-that-inform-me-that-i-have-read-only-role-in-a-project-but-it-appears-everywhere-d)

test instructions:

1. User B (project admin) adds User A as Collector
2. User A enters a CR
3. User B changes role of User A to Read-only before User A can save (easiest way to reproduce is for User A to go offline)
4. User A navigates to any page; this triggers a push and the response contains a collect_records object with an item with 5. status_code: 403 which triggers the toast attached to this card.
5. When use B navigates to a different project page, warning no longer appears
